pdg/src/info.rs

```diff
@@ -0,0 +1,515 @@
+use crate::graph::{Graph, GraphId, Graphs, Node, NodeId, NodeKind};
+use itertools::Itertools;
+use rustc_index::vec::IndexVec;
+use rustc_middle::mir::Field;
+use std::collections::HashMap;
+use std::collections::HashSet;
+use std::cmp;
+use std::fmt::{self, Debug, Display, Formatter};
+
+/// The information checked in this struct is whether nodes flow to loads, stores, and offsets (pos
```

Comment:
Is one node "flowing" to another node semantically the same as there being a directional path from one node to another?

Reply:
Also curious if a node flows to another if it's only a descendant, only and ancestor, or either

Reply:
Yes, node A flows to node B if there's a path from B to A along "source" edges.